Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/SqlServer/SqlDialectTest.cs

```diff
@@ -24,11 +28,15 @@ namespace Remotion.Data.DomainObjects.UnitTests.Persistence.Rdbms.SqlServer
   public class SqlDialectTest
   {
     private SqlDialect _dialect;
+    private Mock<TypeConverter> _typeConverterStub;
+    private StorageTypeInformation _storageTypeInformation;
 
     [SetUp]
     public void SetUp ()
     {
       _dialect = new SqlDialect();
+      _typeConverterStub = new Mock<TypeConverter>();
+      _storageTypeInformation = new StorageTypeInformation(typeof(bool), "test", DbType.Boolean, false, null, typeof(int), _typeConverterStub.Object);
```

Comment:
please create the StorageTypeInfo in each test. we already create a custom instance most of the time and it leads to possible copy&paste bugs / harder readability when mixing fields and variables for the same concept in the test fixture. If you need the same isnance more than once in the test fixture but a special one in other cases, use a factory method for it.